youtokentome/yttm_cli.py

```diff
@@ -110,17 +110,36 @@ def encode(model, output_type, n_threads, bos, eos, reverse, stream):
     bpe.encode_cli(output_type, stream, bos, eos, reverse)
 
 
+def validate_ignore_ids(ctx, param, value):
+    try:
+        if value is not None:
+            return [int(idx) for idx in value.split(",")]
+        else:
+            return None
+    except ValueError:
+        raise click.BadParameter(
+            "Wrong format: expected array of integers, but found {}".format(value)
+        )
+
+
 @click.command()
 @click.option(
     "--model",
     type=click.Path(exists=True),
     required=True,
     help="Path to file with learned model.",
 )
-def decode(model):
+@click.option(
+    "--ignore_ids",
+    type=click.STRING,
+    callback=validate_ignore_ids,
+    required=False,
+    help="Array of indices to ignore during the decoding.",
```

Comment:
It would be better to specify the exact format. For instance, 

> "List of indices to ignore for decoding. Example: --ignore_ids=1,2,3"

. + the same in `README.md`.